deps/spidershim/src/accessor.h

```diff
@@ -240,6 +240,36 @@ JSObject* CreateAccessor(JSContext* cx, CallbackType callback,
   return funObj;
 }
 
+template<class N>
+bool SetAccessor(JSContext* cx,
+                 JS::HandleObject obj,
+                 Handle<N> name,
+                 JSObject* getter,
```

Comment:
getter and setter are gc hazards here: they're raw pointers held across possible GC in JS_WrapValue/JS_ValueToId.

They should be passed in as handles, not raw pointers.